app-drawer/app-drawer.html

```diff
@@ -303,22 +319,24 @@
         this._updateDocScroll();
       },
 
-      _positionChanged: function() {
-        this.fire('app-drawer-position-changed');
-      },
-
       _translateDrawer: function(x) {
         this.transform('translate3d(' + x + 'px, 0, 0)', this.$.contentContainer);
       },
 
       _updateDocScroll: function() {
-        this.shouldEnableDocScroll(this.persistent || !this.opened);
+        var disableScroll = !this.persistent && this.opened;
+        document.body.style.overflow = disableScroll ? 'hidden' : '';
+
+        var appHeaderLayout = document.querySelector('app-header-layout');
```

Comment:
oh I see. This is the main app-header-layout. In that case, we may need to move this to the parent `app-drawer-layout`